com.unity.perception/Editor/Randomization/Utilities/UIElementsEditorUtilities.cs

```diff
@@ -66,7 +67,7 @@ public static PropertyField CreatePropertyField(SerializedProperty iterator, Typ
         {
             var propertyField = new PropertyField(iterator.Copy());
             propertyField.Bind(iterator.serializedObject);
-            var originalField = parentPropertyType.GetField(iterator.name);
+            var originalField = parentPropertyType.GetField(iterator.name, BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance );
             var tooltipAttribute = originalField.GetCustomAttributes(true)
```

Comment:
unrelated to new code, but `originalField` could be null here. Maybe add a null check?

Reply:
Could it? If it is ever null that would be a bug with this code, unless I'm missing something

Reply:
It can happen if there are stale serialized properties in the assets that do not exist in the classes any more, right? Might be a good idea to just ignore them if we come across them.

Reply:
Those fields should be removed from the SerializedObject at import time. Just to be sure, I tested it out by adding a field and then removing it, and I didn't see any issues. Good thing to try though 👍🏻

Reply:
Aa yes, you're right. Neat!